docs/source/notebooks/wps_split_merged_climos_demo.ipynb

```diff
@@ -0,0 +1,173 @@
+       "    NetCDF files to process\n",
+       "loglevel : {'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'}string\n",
+       "    Logging level\n",
+       "\n",
+       "Returns\n",
+       "-------\n",
+       "output : ComplexData:mimetype:`application/metalink+xml; version=4.0`\n",
+       "    Metalink object between output files\n",
+       "\u001b[0;31mFile:\u001b[0m      ~/Desktop/thunderbird/</home/sangwonl/Desktop/thunderbird/venv/lib/python3.6/site-packages/birdy/client/base.py-3>\n",
+       "\u001b[0;31mType:\u001b[0m      method\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "thunderbird.split_merged_climos?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### We can use the docstring to ensure we provide the appropriate parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test local and opendap files\n",
+    "tasmax_climos_local = resource_filename('tests', 'data/tiny_downscaled_tasmax_climos.nc')\n",
+    "hydromodel_climos_opendap = \"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/TestData/tiny_hydromodel_gcm_climos.nc\"\n",
+    "multiple_inputs = [tasmax_climos_local, hydromodel_climos_opendap]\n",
+    "\n",
+    "output = thunderbird.split_merged_climos(multiple_inputs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "req = requests.get(output.get()[0])\n",
+    "split_filepaths = [out_file.contents[0].strip() for out_file in BeautifulSoup(BeautifulSoup(req.content.decode('utf-8')).prettify()).find_all('metaurl')]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Once the process has completed we can extract the results and ensure it is what we expected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper function copied from test_split_merged_climos.py of ce-dataprep\n",
+    "# https://github.com/pacificclimate/climate-explorer-data-prep/blob/master/tests/test_split_merged_climos.py#L13\n",
+    "\n",
+    "def basename_components(filepath):\n",
+    "    \"\"\"Returns the CMOR(ish) components of the basename (filename) of the given filepath.\"\"\"\n",
+    "    # Slightly tricky because variable names can contain underscores, which separate components.\n",
+    "    # We find the location f of the frequency component in the split and use it to assemble the components properly.\n",
+    "    pieces = os.path.basename(filepath).split('_')\n",
+    "    f = next(i for i, piece in enumerate(pieces) if 'Clim' in piece)\n",
+    "    return ['_'.join(pieces[:f])] + pieces[f:]"
+   ]
```

Comment:
I don't think this is a bad test, but I don't love that we are copying this existing method over from another test set. Perhaps we can dumb down this test by simply checking to see if the output contains the expected number of files?